src/TwillBlocks.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace A17\Twill;
+
+use A17\Twill\Services\Blocks\Block;
+use A17\Twill\Services\Blocks\BlockCollection;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\File;
+
+class TwillBlocks
+{
+    public const DIRECTORY_TYPE_VENDOR = 'vendor';
+
+    /**
+     * @var array<string, string>
+     */
+    public static $blockDirectories = [];
+
+    /**
+     * @var array<string, string>
+     */
+    public static $repeatersDirectories = [];
+
+    /**
+     * @return A17\Twill\Services\Blocks\BlockCollection
+     */
+    private $blockCollection;
+
+    /**
+     * Registers a blocks directory.
```

Comment:
This solves the biggest TODO from https://github.com/area17/twill/pull/1306
- publish block and repeater Blade templates

2 other minor limitations are:
1. publish block Vue components
2. publish other types of assets (styles, icons)

IMHO, `1.` is propably not even necessary because block Vue components are not as useful as they seem. And `2.` would be a nice to have, mostly to add the ability to publish custom icons for block types.